Simplified/Book/UI/NYPLBookCellDelegate.m

```diff
@@ -237,46 +237,11 @@ - (void)openAudiobook:(NYPLBook *)book withJSON:(NSDictionary *)json decryptor:(
       DefaultAudiobookManager *const manager = [[DefaultAudiobookManager alloc]
                                                 initWithMetadata:metadata
                                                 audiobook:audiobook];
-      manager.refreshDelegate = self;
```

Comment:
this was just cut and pasted into the new function `createPlayerVCForAudiobook:withBook:configuringAudiobookManager:`